### PR TITLE
feat: station map

### PR DESCRIPTION
**Pull request by chayanforyou**
#4 As per previous discussion, I've implemented the station map which bacially loads .webp (optimized than png) images. I also tried to add both the Bangla and English version of the map.

Please take a look.

https://github.com/user-attachments/assets/89a67820-5bb2-4961-91e3-75ce3ff31121

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/ui/screens/more/MoreScreen.kt [hunk 1]
```diff
@@ -83,7 +89,7 @@ fun MoreScreen(
         verticalArrangement = Arrangement.Top
     ) {
         Column {
-            SectionHeader(text = stringResource(Res.string.settings))
+            //SectionHeader(text = stringResource(Res.string.settings))
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/ui/screens/more/MoreScreen.kt:92]: Can we uncomment this back please?

**Review by aniruddha-adhikary — COMMENTED** [review]
This looks amazing @chayanforyou 

Do you mind restoring the "Settings" header on the main screen? 🙏🏽

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/ui/screens/more/MoreScreen.kt (hunk 1 above)
Comment (chayanforyou) [reply composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/ui/screens/more/MoreScreen.kt]: Sure, We can do it. Please check now.

**Review by aniruddha-adhikary — APPROVED** [review]
Looks good, simplistic implementation to get us off the ground.

**Pull request merged**